### PR TITLE
Add ClippingPlaneCollection add/remove events

### PR DESCRIPTION
**Pull request by hpinkos**
This adds a `planeAdded` and `planeRemoved` events to `ClippingPlaneCollection`

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @hpinkos!
* :heavy_check_mark: Signed CLA found.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by hpinkos** [comment]
@mramato can you review please?

File: Source/Scene/ClippingPlaneCollection.js [hunk 1]
```diff
@@ -149,6 +153,18 @@ define([
             }
         },
 
+        /**
+         * Gets the event fired when members of the collection have been added or removed
+         * @type {Event}
+         * @memberof ClippingPlaneCollection.prototype
+         * @readonly
+         */
+        collectionChanged : {
```
Comment (mramato) [inline Source/Scene/ClippingPlaneCollection.js]: This should probably follow the same pattern as other objects in Scene, such as ImageryLayerCollection/ModelAnimationCollection, which have specific `layerAdded`/`layerRemoved` events. We should also pass the plane and index as part of the event.

Comment (hpinkos) [reply Source/Scene/ClippingPlaneCollection.js]: 👍 good suggestion thanks

**Comment by hpinkos** [comment]
@mramato this is ready

**Comment by mramato** [comment]
Thanks.

**Comment by mramato** [comment]
@hpinkos I was quick on the draw and just realized CHANGES needs to be updated to reflect the review changes and also mention the braking change.

**Comment by mramato** [comment]
(Please just do it in master)

**Comment by hpinkos** [comment]
Breaking change?  There is no breaking change in this PR

**Comment by hpinkos** [comment]
Update CHANGES https://github.com/AnalyticalGraphicsInc/cesium/commit/bb856df917684e570d0fc688a4acf285bd878f1e

**Comment by mramato** [comment]
> Breaking change? There is no breaking change in this PR

Yeah, I got it confused with your clone PR.

**Pull request merged**